README.md

```diff
@@ -246,6 +247,12 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [zeromq-crystal](https://github.com/benoist/zeromq-crystal) - Bindings for ZeroMQ
  * [zlib.cr](https://github.com/datanoise/zlib.cr) - Bindings for ZLib library
 
+## Markdown/Text Processors
+
+ * [HTML::Pipeline](https://github.com/huacnlee/html-pipeline) - HTML processing filters and utilities for Crystal.
+ * [Markdown](https://crystal-lang.org/api/latest/Markdown.html) - Markdown library parses Markdown text in Crystal Stdlib.
+ * [Remarkdown](https://github.com/huacnlee/remarkdown) - This is extends of Markdown lib from Crystal Stdlib for Support Markdown GFM.
```

Comment:
Wouldn't it be better to reformulate it somehow:

`This is an extension for Crystal markdown that supports GFM`

or just

`GitHub Flavored Markdown for Crystal`